release-team/role-handbooks/bug-triage/README.md

```diff
@@ -97,6 +100,45 @@ Before starting, the Bug Triage members should refer to the following guides to
 - [the documentation for issue `kind` labels](https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label)
 - [the documentation for defining priority and `priority` labels](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#step-three-define-priority).
 
+### Bug Triage Project Board
+
+The Bug Triage team is using a [GitHub project board](https://github.com/orgs/kubernetes/projects/80) to track the current status of all issues and PRs targeting the release, their priorities, and to distribute the work among the Bug Triage team.
+
+Bug triage team members are expected to assign themselves as the `Responsible` person to track an issue to the extent of their time availability throughout the cycle. The bug triage team should keep an eye on the board for new issues/PRs and make sure they are tracked.
+
+The project board is public so that both the release team and anyone interested from the community can stay updated about the current status of issues and PRs that are targetting the release. Write access to the board is limited to members of [@kubernetes/release-team-bug-triage](https://github.com/orgs/kubernetes/teams/release-team-bug-triage).
+
+New issues and PRs that target the current release milestone are automatically added to the board with a [a prow job](https://prow.k8s.io/?job=periodic-sync-bug-triage-github-project-*). The prow job is defined in [kubernetes/test-infra](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml) and the script can be found in [kubernetes/sig-release](https://github.com/kubernetes/sig-release/blob/master/release-team/hack/sync-bug-triage-github-project-beta.sh).
+
+#### Information on the Bug Triage Board
+
+For each board item, the following details are set and managed by the bug triage team:
+
+- `Responsible`: The bug triage team member that is responsible for tracking the progress on the issue/PR. This is not the same as the issue assignee
+- `Notes`: Short note by the bug triage team member regarding the issue/PR, e.g. `[6/3] No progress made since last comment; notified them about the code freeze`
+- `Status`: One of the following values:
+  - `Tracked`: a member of the bug triage team is actively tracking the issue
+  - `Pending inclusion`: the issue is not actively tracked by anyone
+
+The following details for each item can be seen on the board. Note that it is not possible to edit these fields from the board directly:
+
+- `Title`: Issue/PR title
+- `Type`: Issue or Pull Request
```

Comment:
this seems to be editable even though it makes sense to make it uneditable if there is a way